src/tcutility/analysis/report/report.py

```diff
@@ -0,0 +1,111 @@
+import pathlib as pl
+from typing import Union
+
+import docx
+from htmldocx import HtmlToDocx
+from tcutility.analysis.report.formatters.generic import WordFormatter
+from tcutility.analysis.report.formatters.xyz import XYZFormatter
+from tcutility.results import Result, read
+
+
+class SI:
+    def __init__(self, path: Union[str, pl.Path], append_mode: bool = False, font: str = "Arial", format: WordFormatter = XYZFormatter()) -> None:
+        """Initializes the SI class for creating supporting information (SI) files in Microsoft Word format.
+
+        This class is responsible for creating and managing a Microsoft Word document that serves as supporting information (SI) for reports or publications.
+        It allows for the addition of various elements such as text, headings, and formatted content from HTML.
+
+        Args:
```

Comment:
arg-types will automatically be taken from the annotations and then written to .rst by sphinx, so only the argument names are necessary here.

Reply:
Aha, good to know!